### PR TITLE
chore(docs): replace legacy blockquotes with component

### PR DESCRIPTION
**Pull request by krisantrobus**

### CODE REVIEW

File: packages/paste-website/src/pages/components/pagination/index.mdx [hunk 1]
```diff
@@ -343,16 +343,15 @@ Most Twilio datasets show 10 or 50 rows per page by default. However, a generic
...
-  of trying to decide their display preference.
+<Blockquote url="https://www.nngroup.com/articles/item-list-view-all/">
+  <BlockquoteContent>
+    Many sites let users choose how many items they’ll see on each page. This is often overkill...It’s usually better to
+    offer a single default number — such as 10 or 20 — [or] give users the choice between two numbers, say 10 and 50,
+    where the second number is substantially bigger than the default. If the choice is between two relatively similar
+    numbers (such as 10 and 20), users might as well click the Next Page button rather than suffer the cognitive
+    overhead of trying to decide their display preference.
+  </BlockquoteContent>
+  <BlockquoteCitation author="Nielsen Norman Group" source={`Users' Pagination Preferences and "View All"`} />
```
Comment (krisantrobus) [inline packages/paste-website/src/pages/components/pagination/index.mdx:354]: Changed the source and author as other quotes form this website use the group as the author. Changed for consistency across the site

File: packages/paste-website/src/pages/patterns/filter/index.mdx [hunk 2]
```diff
@@ -81,12 +81,11 @@ This pattern provides recommendations for selection methods, layout, and behavio
...
       <ListItem>Map out the relationships between different filters and choose the ones that will be part of your feature. Remember filters should give users a sense of control and not overwhelm them.</ListItem>
-      <Blockquote
-        name="Defining Helpful Filter Categories and Values for Better UX"
-        source="Nielsen Norman Group"
-        sourceUrl="https://www.nngroup.com/articles/filter-categories-values/"
-      >
-        A truly usable faceted search provides filter categories and filter values that are appropriate, predictable, free of jargon, and prioritized
+      <Blockquote url="https://www.nngroup.com/articles/filter-categories-values/">
+        <BlockquoteContent>
+              A truly usable faceted search provides filter categories and filter values that are appropriate, predictable, free of jargon, and prioritized
```
Comment (nkrantz) [inline packages/paste-website/src/pages/patterns/filter/index.mdx:86]:
```suggestion
              A truly usable faceted search provides filter categories and filter values that are appropriate, predictable, free of jargon, and prioritized.
```

nit: let's just make this change if any other changes are needed - not necessary to do another commit just for this

**Review by nkrantz — COMMENTED** [review]
looks great! let's also remove the old codeblock component since we don't need it anymore.

**Review by PixeledCode — APPROVED** [review]

**Review by jhanvi1399 — APPROVED** [review]
